### PR TITLE
Hw 03

### PR DESCRIPTION
**Pull request by kravets-julia**
Перевірте, будь ласка :)

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const DB_HOST =
+  "mongodb+srv://Julia:GMBgtOeL6YD9JsJS@cluster0.p4bupmv.mongodb.net/db-contacts?retryWrites=true&w=majority";
```
Comment (ceREAL133) [inline config.js]: connection string should be hidden inside .env

File: server.js [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+    app.listen(PORT, () => {
+      console.log("Database connection successful");
+    });
+  })
+  .catch((error) => {
+    console.log(error.message);
+    process.exit(1);
+  });
+
+// app.listen(3000, () => {
```
Comment (ceREAL133) [inline server.js]: delete comment

File: models/contact.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      type: Boolean,
+      default: false,
+    },
+  },
+  {versionKey: false, timestamps: true}
+);
+
+const addSchema = Joi.object({
+  name: Joi.string().required(),
+  email: Joi.string().required(),
```
Comment (ceREAL133) [inline models/contact.js]: this schema should validate email and phone. Your schema only validates string

**Comment by kravets-julia** [comment]
corrected comments

**Review by ceREAL133 — APPROVED** [review]

**Pull request merged**